src/util.js

```diff
@@ -43,6 +43,10 @@ export const recursiveMerge = (a, b) => {
     return ret;
 };
 
+export const uniquify = (values) => {
+    return [...new Set(values)];
```

Comment:
Yeah we're on the same page. Since outputting unique `font-family`s isn't important we can skip all of this.